typescript-conditionals/main.ts

```diff
@@ -0,0 +1,123 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro,
+            recommendMovie
+ */
+
+const isUnderFive = (number: number): boolean => number < 5;
+console.log('Test(4):', isUnderFive(4));
+// true
+console.log('Test(10):', isUnderFive(10));
+// false
+console.log('Test(5):', isUnderFive(5));
+// false
+
+const isEven = (num: number): boolean => num % 2 === 0;
+
+console.log('Test isEven 4', isEven(4));
+// true
+console.log('Test isEven 10', isEven(10));
+// true
+console.log('Test isEven 5', isEven(5));
+// false
+const startsWithJ = (string: string): boolean => string[0] === 'J';
+console.log('startsWithJ JavaScript', startsWithJ('JavaScript'));
+// true
+console.log('startsWithJ PHP', startsWithJ('PHP'));
+// false
+console.log('startsWithJ HTML', startsWithJ('HTML'));
+// false
+console.log('startsWithJ Java', startsWithJ('Java'));
+// true
+
+const isOldEnoughToDrive = (person: { name: string; age: number }): boolean =>
+  person.age > 16;
+const homer = {
+  name: 'Homer Jay Simpson',
+  age: 39,
+};
+
+console.log(isOldEnoughToDrive(homer));
+
+const isOldEnoughToDrinkAndDrive = (person: {
+  name: string;
+  age: number;
+}): boolean => {
+  console.log(person); // ？？？
+  return false;
+};
```

Comment:
Since these functions take the exact same object, you should create an `interface` for the object instead and use that for your type annotation

```suggestion
interface Person {
  name: string;
  age: number;
 }

const isOldEnoughToDrive = (person: Person): boolean => person.age > 16;
  
const homer: Person = {
  name: 'Homer Jay Simpson',
  age: 39,
};

console.log(isOldEnoughToDrive(homer));

const isOldEnoughToDrinkAndDrive = (person: Person): boolean => {
  console.log(person); // ？？？
  return false;
};
```